lib/collection/src/operations/verification/query.rs

```diff
@@ -35,6 +35,52 @@ impl Query {
         }
         Ok(())
     }
+
+    /// Check that the query does not perform a fullscan based on the collection configuration.
+    async fn check_fullscan(
+        &self,
+        using: &str,
+        collection: &Collection,
+        strict_mode_config: &StrictModeConfig,
+    ) -> CollectionResult<()> {
+        // Check only applies on `unindexed_filtering_retrieve`
```

Comment:
Shouldn't it be `search_allow_exact` instead?

Reply:
Good catch!
You are right, got confused with my ongoing work on group by filtering :facepalm: